File: src/MovieTicket.java
public class MovieTicket {
    int rowNr;
    int seatNr;
    boolean isPremium;
    MovieScreening movieScreening;

    public MovieTicket(MovieScreening movieScreening, int rowNr, int seatNr, boolean isPremium){
        this.isPremium = isPremium;
        this.rowNr = rowNr;
        this.seatNr = seatNr;
        this.movieScreening = movieScreening;

    }

    public boolean isPremium() {
        return isPremium;
    }
    public double getPrice(){
        double price;
        if (isPremium = false){
            price = movieScreening.pricePerSeat;
        } else {
            price = movieScreening.pricePerSeat + 2;
        }
        return price;
    }

    @Override
    public String toString() {
        return "MovieTicket [rowNr=" + rowNr + ", seatNr=" + seatNr + ", isPremium=" + isPremium + "]";
    }
}

File: Domain/MovieTicket.cs
namespace Domain
{
    public class MovieTicket
    {
        private int _rowNr { get; set; }
        private int _seatNr { get; set; }
        public bool IsPremium { get; set; }
        public MovieScreening MovieScreening { get; set; }

        public MovieTicket(int rowNr, int seatNr, bool isPremium, MovieScreening movieScreening)
        {
            _rowNr = rowNr;
            _seatNr = seatNr;
            IsPremium = isPremium;
            MovieScreening = movieScreening;
        }

        public double GetPrice()
        {
            return MovieScreening.GetPricePerSeat();
        }
    }
}